clang/test/SemaSYCL/deferred-diagnostics-aux-builtin.cpp

```diff
@@ -1,16 +1,8 @@
-// RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -fsycl -fsycl-is-device -aux-triple x86_64-unknown-linux-gnu -Wno-sycl-2017-compat -verify -fsyntax-only  %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown-sycldevice -aux-triple x86_64-unknown-linux-gnu -Wno-sycl-2017-compat -verify -fsyntax-only  %s
```

Comment:
Should this have -sycl-std=2020

Reply:
This test did not throw the warning `Passing of kernel functions by reference is a SYCL 2020 extension` which I why I did not add the `-sycl-std=2020` flag. But I can add one if we need to be consistent with other tests or keep the standard to 2020 by default.

Reply:
I think this is fine.